src/using-document-creator.mdx

```diff
@@ -0,0 +1,179 @@
+  "defaults": {...},
+  "schema": {...},
+  "attachments": {...}
+}
+```
+
+### `"name"` field
+
+The `name` field is a string that refers to the name of the document.
+
+### `"type"` field
+
+The `type` field is a string that reters to the type of the document, either `"TRANSFERABLE_RECORD"` or `"VERIFIABLE_DOCUMENT"`.
+
+### `"defaults"` field
+
+The `defaults` field is an object that contains the settings that you have that are constant among the different documents of the same kind of file.
+
+Usually the document store address, token registry address, template, issuers, company logo, signatures, etc. are stored here, since they are a constant in the different documents of the same kind.
+
+An example of the `"defaults"` field:
+
+```json
+"defaults": {
+  "$template": {
+    "type": "EMBEDDED_RENDERER",
+    "name": "BILL_OF_LADING",
+    "url": "https://demo-cnm.openattestation.com"
+  },
+  "issuers": [
+    {
+      "identityProof": {
+        "type": "DNS-TXT",
+        "location": "isaactest.xyz"
+      },
+      "name": "Demo Issuer",
+      "tokenRegistry": "0x6D489a6A2cbcB76FcEeA26102d1273cd4d3B6b10"
+    }
+  ],
+  "name": "Demo bill of lading"
+}
+```
+
+#### `"$template"` field
+
+The `$tempalte` field is an object that contains information about the [custom rendering template](./custom-template).
+
+#### `"issuers"` field
+
+The `issuers` is an array of the issuer object. The required fields for each issuer are `name`, `identityProof` and one of `documentStore` OR `tokenRegistry`.
+
+Transferrable document will use a [`tokenRegistry`](./token-registry), where as, a verifiable document will use a `documentStore`.
+
+The `name` field is a string that refers to the name of the issuer.
+
+The `tokenRegistry` field is a string that refers to the address in which you deploy the token registry.
+
+The `documentStore` field is a string that refers to the address in which you deploy the document store.
+
+The `identityProof` field is an object that requires the field `type` and `location`.
+
+The `type` field is a string that refers to the type of configuration you have set up.
+
+The `location` field is a string that refers to the domain url in which you have set up the configured DNS.
+
+### `"schema"` field
+
+For the schema we are using JSON schema to populate the form.
+
+The construction of the schema is defived from when you are constructing your custom renderer.
+
+The fields in here must match the one in your specific custom renderer.
+
+If not it will now display on the custom renderer.
+
+An example of the `"schema"` field:
+
+```json
+"schema": {
+  "type": "object",
+  "required": ["blNumber"],
+  "properties": {
+    "blNumber": {
+      "type": "string",
+      "title": "BL number"
+    },
+    "item": {
+      "type": "string",
+      "title": "Goods"
+    }
+  }
+}
+```
+
+### `"Attachments"` field
+
+This `"attachments"` field contains the information about the required attachments for this document.
```

Comment:
this make it looks like adding the attachments property will force any document to have attachments (but I dont think it's the case)

Reply:
ok